src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppManagement.java

```diff
@@ -56,4 +58,98 @@ public static void disableAppAutoRestart() {
     Logger.LOG_DEBUG("Disabling next JVM run command.");
     RuntimeControl.configureNextRunCommand(jvmCommand);
   }
+
+  /**
+   * Checks if a valid WAN IP address has been assigned to the device.
+   *
+   * @return true if an initialized WAN IP address is found.
+   */
+  public static boolean deviceHasWanIP() {
+    boolean hasWanIp = false;
+    final String uninitializedWan = "0.0.0.0";
```

Comment:
```suggestion
    final String uninitializedWanIp = "0.0.0.0";
```
This variable could be named a bit more consistently as it is compared with .getWanIp()